omim2obo/main.py

```diff
@@ -397,18 +417,9 @@ def omim2obo(use_cache: bool = False):
             if len(assocs) > 1 or not p2g_is_definitive(p_lab):  # or cases above: (p_map_key != '3') & p_mim_excluded
                 continue
 
-            # Log review.tsv cases
             log_review_cases(p_mim, p_lab, p_map_key, gene_mim, gene_phenotypes, omim_types)
-
-            # Add restrictions: Disease-defining ('causal germline mutation')
-            # - Disease --(RO:0004003 'has material basis in germline mutation in')--> Gene
-            #   https://www.ebi.ac.uk/ols4/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0004003
-            add_subclassof_restriction_with_evidence_and_source(
-                graph, RO['0004003'], OMIM[gene_mim], OMIM[p_mim], evidence)
-            # - Gene --(RO:0004013 'is causal germline mutation in')--> Disease
-            #   https://www.ebi.ac.uk/ols4/ontologies/ro/properties?iri=http://purl.obolibrary.org/obo/RO_0004013
-            add_subclassof_restriction_with_evidence_and_source(
-                graph, RO['0004013'], OMIM[p_mim], OMIM[gene_mim], evidence)
+            # TODO: move above && use at end of this block, & at beginning of block if in inclusions
```

Comment:
Please implement any reasonably easy TODOs rather than leaving comments.

Reply:
Oh sorry I am not finished this this PR yet. that TODO is not suppose dto be there any longer. I hadn't tagged you yet on tis.